### PR TITLE
some bug fixes

### PR DESCRIPTION
**Pull request by zzlei**
Also remove the use of bzero, which is deprecated.

### CODE REVIEW

File: src/jumbo.c [hunk 1]
```diff
@@ -262,7 +262,7 @@ int strcasecmp(char *dst, char *src) {
 		if ( ((l = (unsigned char)(*(src++))) >= 'A') && (l <= 'Z') )
 			l -= 'A' - 'a';
 	} while (f && (f==l));
-	return return(f - l);
```
Comment (magnumripper) [inline src/jumbo.c]: Wow. How did that pass?

**Comment by magnumripper** [comment]
Was there a bug fix in there besides the `return return`? I'm not saying you did anything wrong, just trying to understand. The bzero crap was already not used anymore, right?

**Comment by zzlei** [comment]
Yeah, I found no bug with bzero indeed. I just think it's deprecated and removing them should cause no trouble.

File: src/jumbo.c (hunk 1 above)
Comment (zzlei) [inline src/jumbo.c]: I was amazed, too. At first I even thought it was some arcane syntax.

**Pull request merged**